Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/ArcGISSendBinding.cs

```diff
@@ -349,7 +343,7 @@ public async Task Send(string modelCardId)
           var result = await unitOfWork.Service
             .Execute(
               mapMembers,
-              sendInfo,
+              modelCard.GetSendInfo("ArcGIS"), // POC: get host app name from settings?
```

Comment:
okay we will need to resolve this at one point :D (just a comment)

Reply:
yes, I'm fairly certain we have a ticket for this, but it was always low prio